ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -874,23 +874,21 @@ const triggerSet: TriggerSet<Data> = {
         const defamation = data.defamationColor;
         if (defamation === undefined)
           return;
-        if (regression === 'remote') {
-          const color = defamation === 'red' ? output['blue']!() : output['red']!();
-          return output.nearTether!({ color: color });
-        }
+        const color = defamation === 'red' ? output.blue!() : output.red!();
+        if (regression === 'remote')
+          return output.farTether!({ color: color });
 
         if (parseFloat(matches.duration) < 80)
-          return output.farTether!({ color: output[defamation]!() });
+          return output.nearTether!({ color: color });
 
-        const color = defamation === 'red' ? output['blue']!() : output['red']!();
         return output.finalTowerFar!({ color: color });
       },
       outputStrings: {
-        nearTether: {
+        farTether: {
```

Comment:
Other places in this file treat "far tether" as "tether that will break when far", so making this consistent.

Reply:
Makes sense to me. I looked at other matches for these names to try and get it to be consistent, but it appears I made a mistake.